src/components/edit-user.html

```diff
@@ -55,29 +55,22 @@ <h1>Edit User</h1>
 
   <script type="module">
     import { Element as PolymerElement } from '@banno/polymer/polymer-element.js';
-    
+
     class EditUser extends PolymerElement {
       static get is() { return 'edit-user'; }
       static get properties() {
         return {
-          storage: Object,
+          database: Object,
           profileToEdit: Object,
-          profileIndex: Number,
-          editedProfile: {
-            type: Object,
-            value() {
-              return {};
-            }
-          }
         }
       }
-      
+
       submitChanges() {
-        this.storage.updateUser(this.editedProfile, this.profileIndex);
+        this.database.ref('Profiles/' + this.profileToEdit.userId).set(this.profileToEdit);
```

Comment:
should use template strings here

Reply:
```const profId = this.profileToEdit.userId;
      this.database.ref(`Profiles/ ${profId}`).set(this.profileToEdit);```
Is this what you meant? It's not working ...

Reply:
you're missing the [ticks](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#Syntax):
```suggestion
        this.database.ref(`Profiles/${this.profileToEdit.userId}`).set(this.profileToEdit);
```